.github/workflows/release.yml

```diff
@@ -131,13 +150,39 @@ jobs:
           echo "$promotion_summary :frog: Version ${{ inputs.release_version }} was promoted to ${rt_repo}" >> $GITHUB_STEP_SUMMARY
 
       - name: Checkout reusable workflow source repo
-        if: ${{ inputs.box_upload == true }}
         uses: actions/checkout@v4
         with:
           repository: pentaho/actions-common
           path: actions-common
-          ref: stable
+          ref: HNC-539
 
+      - name: Create Release Bundle
+        if: ${{ inputs.create_release_bundle == true }}
+        shell: bash
+        run: |
+          python3 -m venv env
+          source env/bin/activate
+          pip3 install -q pyyaml dohq-artifactory requests boxsdk tqdm
```

Comment:
If this is a repeating requirement, this needs to be added to the bootstrap image.

Reply:
Ok I will add this in the bootstrap image

Reply:
The [Dockerfile](https://github.com/pentaho/actions-common/pull/18/files#diff-7ca2f7aa06466baf070ceb935ad071456cbca0092d65747c0712df8130aaed42R22) still has the inclusion of theses libs. Keep them only in one place then.